codeexamples/index.html

```diff
@@ -7,14 +7,22 @@
     <div id="root">
     </div>
     <div>
+      <p>These examples output to the javascript console. To view the output, open developer tools (F12) and examine the javascript console.</p>
```

Comment:
Nice!

Reply:
I can see these are very much WIP, and I suspect over time developers may find it easier to work with a working example site built to work with the JS library - with working rendered results etc

It may even contribute towards the take up of your engine :-) 

(which is very good by the way - I'm currently integrating this for my client to go live next week)

Reply:
Yes, the code samples and /public are both very much WIP. A full-blown example web app would be great, but we haven't had time to add that yet. 
Thank you for the compliment on the engine. Let me know if you have improvement suggestions for that as well!